apps/internal-ui/src/components/Dialog/FormDialog/FormDialog.tsx

```diff
@@ -0,0 +1,45 @@
+import clsx from 'clsx';
+
+import { FormDialogProps } from '@/components/Dialog/FormDialog/types';
+import Dialog from '@/components/Dialog/shared/Dialog';
+import OverlayTitle from '@/components/shared/components/Overlay/OverlayTitle';
+import { DIALOG_OVERLAY_CONTENT_WRAPPER_ELEMENTS } from '@/components/shared';
+
+const FormDialog = ({
+  children,
+  isOpen,
+  ref,
+  title,
+  className,
+  innerClassName,
+  onCancel,
+  onConfirm,
+  confirmButtonLabel,
+  cancelButtonLabel,
+  possibleConfirm,
+  isPending,
+}: FormDialogProps) => {
+  return (
+    <Dialog isOpen={isOpen} ref={ref} className={className}>
+      <Dialog.Header>
+        <OverlayTitle title={title} />
+      </Dialog.Header>
+      <Dialog.ContentWrapper
+        as={DIALOG_OVERLAY_CONTENT_WRAPPER_ELEMENTS.FORM}
+        className={clsx(innerClassName, 'gap-y-6')}
```

Comment:
p3: 단순히 inner보다는 저희가 header, content, footer라는 영역으로 나눠놨으니 이를 좀 더 적극적으로 활용해서 contentClassName 네이밍은 어떨까요?
```suggestion
        className={clsx(contentClassName, 'gap-y-6')}
```

Reply:
구두) [논의 내용](https://bbodek.slack.com/archives/C0663GKH4H3/p1750290766502859)에 따라 innerClassName props 는 제거한다